WalletWasabi/WabiSabi/Client/CoinJoin/Client/CoinJoinCoinSelector.cs

```diff
@@ -128,24 +128,6 @@ public ImmutableList<TCoin> SelectCoinsForRound<TCoin>(IEnumerable<TCoin> coins,
 		// Shuffle coins, while randomly biasing towards lower AS.
 		var orderedAllowedCoins = AnonScoreTxSourceBiasedShuffle(allowedCoins).ToArray();
 
-		// If the command is given to not stop when everything is coinjoined and the allowed private coins are empty, then we shortcircuit the selection.
-		if (!stopWhenAllMixed && allowedNonPrivateCoins.Count == 0)
```

Comment:
This code was added for being able to select coins, even if all the coins in the wallet are private. We do not need that functionality, the reason mentioned above.